R/get_match_report.R

```diff
@@ -8,8 +8,8 @@
 #'
 #' @importFrom magrittr %>%
 #' @importFrom rlang .data
+#' @noRd
```

Comment:
`build_reference()` puts thedot functions into the function reference by default since there are `.Rd` files for them. We don't want them to be in the index.

Reply:
Awesome - thanks for doing this